dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ServerMojos.scala

```diff
@@ -0,0 +1,91 @@
+package com.lightbend.lagom.maven
+
+import javax.inject.Inject
+
+import com.lightbend.lagom.core.LagomVersion
+import com.lightbend.lagom.dev.Servers
+import org.apache.maven.plugin.AbstractMojo
+import org.eclipse.aether.artifact.DefaultArtifact
+import java.util.{ Collections, List => JList, Map => JMap }
+
+import org.codehaus.plexus.logging.{ Logger, LoggerManager }
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.beans.BeanProperty
+
+class StartCassandraMojo @Inject() (facade: MavenFacade, logger: MavenLoggerProxy, mavenLoggerManager: LoggerManager) extends AbstractMojo {
+
+  @BeanProperty
+  var cassandraMaxBootWaitingSeconds: Int = _
+  @BeanProperty
+  var cassandraPort: Int = _
+  @BeanProperty
+  var cassandraEnabled: Boolean = _
+  @BeanProperty
+  var cassandraCleanOnStart: Boolean = _
+  @BeanProperty // I'm not sure if it's possible to specify a default value for a literal list in plugin.xml, so specify it here.
+  var cassandraJvmOptions: JList[String] = Seq("-Xms256m", "-Xmx1024m", "-Dcassandra.jmx.local.port=4099",
+    "-DCassandraLauncher.configResource=dev-embedded-cassandra.yaml").asJava
+
+  override def execute(): Unit = {
+    if (cassandraEnabled) {
+      // Configure logging to quieten the Cassandra driver
+      mavenLoggerManager.getLoggerForComponent("com.datastax").setThreshold(Logger.LEVEL_DISABLED)
+
+      val cp = facade.resolveArtifact(new DefaultArtifact("com.lightbend.lagom", "lagom-cassandra-server_2.11",
```

Comment:
Actually, lagom-cassandra-server doesn't even depend on Scala (or at least it shouldn't), since it's just some configuration files and a dependency on cassandra itself. What we should probably do is turn off cross versioning for it.